lambdas/generator/handler.py

```diff
@@ -158,34 +128,33 @@ def get_important_file_urls(user_name, repo_name, important_file_names):
     return file_paths
 
 
-def get_default_branch(user_name, repo_name):
-    # GitHub API
-    url = "https://api.github.com/repos/" + user_name + "/" + repo_name
-    res = requests.get(url, headers={'Authorization-Type': "Bearer " + os.environ['GITHUB_TOKEN'], "X-GitHub-Api-Version": "2022-11-28"})
+def get_default_branch(github_username, repo_name):
+    url = "https://api.github.com/repos/" + github_username + "/" + repo_name
+    res = requests.get(url, headers=github_api_headers)
 
     data = json.loads(res.content)
     return data['default_branch']
 
 
 def is_important_file(path, important_file_names):
-    # get files with names in important_file_names, and only contains ~2 / (not in a subdirectory)
+    # get files with names in important_file_names, and only contains ~ 1 / (not in a subdirectory)
     for file_name in important_file_names:
         if file_name in path and path.count('/') <= 1:
             return True
     return False
 
 
-def get_file_contents(github_user_name, repository_name, file_path):
-    url = "https://api.github.com/repos/" + github_user_name + "/" + repository_name + "/contents/" + file_path
-    res = requests.get(url, headers={'Authorization-Type': "Bearer " + os.environ['GITHUB_TOKEN'], "X-GitHub-Api-Version": "2022-11-28"})
+def get_file_content(github_username, repository_name, file_path):
+    url = "https://api.github.com/repos/" + github_username + "/" + repository_name + "/contents/" + file_path
+    res = requests.get(url, headers=github_api_headers)
 
     data = json.loads(res.content)
     content_encoded = data['content']
     content = str(base64.b64decode(content_encoded))
     return content
 
 
-def get_criteria(prompt, languages):
+def get_criteria_keywords(prompt, languages):
```

Comment:
Would be helpful if you can add a description explaining what the output from chatgpt api will look like